test/bdd/go.mod

```diff
@@ -0,0 +1,7 @@
+// Copyright SecureKey Technologies Inc. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+module github.com/hyperledger/aries-framework-go/test
+
+go 1.12
```

Comment:
Do we need this ?

Reply:
this was autogenerated when i did go mod init. Which part you are referring go version ?

Reply:
ok. i was referring to 'go 1.12' line.